schema/types/isolator_linux_specific.go

```diff
@@ -78,6 +95,14 @@ func (l linuxCapabilitiesSetBase) AssertValid() error {
 	return nil
 }
 
+// TODO(lucab): both need to be clarified in spec
+func (l linuxCapabilitiesSetBase) MaxInstances() uint {
+	return ^uint(0)
+}
+func (l linuxCapabilitiesSetBase) Conflicts() []ACIdentifier {
+	return nil
```

Comment:
I opened https://github.com/appc/spec/issues/625 to clarify all those points at once, at this seccomp PR was starting to touch way too many things. If it is ok for you, I'd fix that there.